docs/builder.rst

```diff
@@ -46,13 +55,26 @@ The builder (i.e. ``build()``) expects a dict as the first argument. This dict c
    >>> assert built_manifest == expected_manifest
    >>> assert extended_manifest == expected_manifest
 
+With ``init_manifest()``, which populates "version" with "2" (the only supported EthPM specification version), unless provided with an alternative "version".
+
+.. doctest::
+
+   >>> initialized_manifest = build(
+   ...     init_manifest("owned", "1.0.0"),
+   ... )
+   >>> assert initialized_manifest == expected_manifest
```

Comment:
As code examples go, I think this one would be better if you dropped the `expected_manifest` and variable assignment and allowed the doctest to just test against the return value of `build(...)` as it dumps to the terminal.

```
.. doctest
    >>> build(init_manifest('owned', '1.0.0'))
    {...<the-raw-manifest-dict>...}
```